scripts/install-build-tools.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+echo "Setting up build environment..."
+
+green="\033[0;32m"
+cyan="\033[0;36m"
+end="\033[0m"
+
+set -e
+
+SUDO=''
+if (( $EUID != 0 )); then
+    echo -e "non-root user, sudo required"
+    SUDO='sudo'
```

Comment:
break here as in exit rather than running subcommands with `sudo`?

Reply:
This is erroneous on my part -- I am seeing apt update fail (line 33) because this requires elevated privileges, really what this comment should be is that lines 33 and 34 (may) require sudo as well

Reply:
mmm, agreed; those lines definitely need `$SUDO` added; willco.